### PR TITLE
[Doc] Adjust Sequence of tutorial doc.

### PR DESCRIPTION
**Pull request by crazygao**
# Description

Adjust Sequence of doc site.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: docs/tutorials/index.md [hunk 1]
```diff
@@ -12,28 +12,28 @@ This section contains a collection of flow samples and step-by-step tutorials.
...
 |Rag|[Tutorial: how prompt flow helps on quality improvement](https://github.com/microsoft/promptflow/blob/main/examples/tutorials/flow-fine-tuning-evaluation/promptflow-quality-improvement.md)| This tutorial is designed to enhance your understanding of improving flow quality through prompt tuning and evaluation|
 |Rag|[Develop promptflow copilot](https://github.com/microsoft/promptflow/blob/main/examples/tutorials/develop-promptflow-copilot/develop-promptflow-copilot.md)| In this tutorial, we will provide a detailed walkthrough on creating a RAG-based copilot using the Azure Machine Learning promptflow toolkit|
+|Rag|[Chat with pdf in azure](https://github.com/microsoft/promptflow/blob/main/examples/flows/chat/chat-with-pdf/chat-with-pdf-azure.ipynb)|A tutorial of chat-with-pdf flow that executes in Azure AI|
+|Rag|[Chat with pdf - test, evaluation and experimentation](https://github.com/microsoft/promptflow/blob/main/examples/flows/chat/chat-with-pdf/chat-with-pdf.ipynb)|A tutorial of chat-with-pdf flow that allows user ask questions about the content of a PDF file and get answers|
```
Comment (wangchao1230) [inline docs/tutorials/index.md:36]: this should be ahead of the azure one

File: examples/flex-flows/basic/flex-flow-quickstart-azure.ipynb [hunk 2]
```diff
@@ -282,7 +282,7 @@
    ],
    "category": "azure",
    "section": "Flow",
-   "weight": 10
+   "weight": 1010
```
Comment (wangchao1230) [inline examples/flex-flows/basic/flex-flow-quickstart-azure.ipynb:285]: not manully set, but add programmed

File: docs/tutorials/index.md (hunk 1 above)
Comment (crazygao) [reply docs/tutorials/index.md:36]: OK

File: examples/flex-flows/basic/flex-flow-quickstart-azure.ipynb (hunk 2 above)
Comment (crazygao) [reply examples/flex-flows/basic/flex-flow-quickstart-azure.ipynb:285]: HAHA OK

**Review by wangchao1230 — APPROVED** [review]

**Review by zhengfeiwang — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Review by brynn-code — APPROVED** [review]

**Pull request merged**